app/views/shared/buttons/_column_lists.html.haml

```diff
@@ -1,106 +1,113 @@
-            %i.fa.fa-angle-double-up.fa-lg
-        - if @edit[:new][:fields].length < 2
-          %button.btn.btn-default.disabled
-            %i.fa.fa-angle-up.fa-lg
-        - else
-          - t = _("Move selected fields up")
-          %button.btn.btn-default{:title           => t,
-                                  :remote          => true,
-                                  "data-submit"    => 'column_lists',
-                                  "data-method"    => :post,
-                                  "data-click_url" => {:url => url_for_only_path(:action => 'group_form_field_changed',
-                                                                       :button => 'up',
-                                                                       :id     => @custom_button_set.id || 'new')}.to_json}
-            %i.fa.fa-angle-up.fa-lg
-        - if @edit[:new][:fields].length < 2
-          %button.btn.btn-default.disabled
-            %i.fa.fa-angle-down.fa-lg
-        - else
-          - t = _("Move selected fields down")
-          %button.btn.btn-default{:title           => t,
-                                  :remote          => true,
-                                  "data-submit"    => 'column_lists',
-                                  "data-method"    => :post,
-                                  "data-click_url" => {:url => url_for_only_path(:action => 'group_form_field_changed',
-                                                                       :button => 'down',
-                                                                       :id     => @custom_button_set.id || 'new')}.to_json}
-            %i.fa.fa-angle-down.fa-lg
-        - if @edit[:new][:fields].length < 2
-          %button.btn.btn-default.disabled
-            %i.fa.fa-angle-double-down.fa-lg
-        - else
-          - t = _("Move selected fields to bottom")
-          %button.btn.btn-default{:title           => t,
-                                  :remote          => true,
-                                  "data-submit"    => 'column_lists',
-                                  "data-method"    => :post,
-                                  "data-click_url" => {:url => url_for_only_path(:action => 'group_form_field_changed',
-                                                                       :button => 'bottom',
-                                                                       :id     => @custom_button_set.id || 'new')}.to_json}
-            %i.fa.fa-angle-double-down.fa-lg
+  - if x_active_tree == :ab_tree
+    .col-md-5
+      = _('Unassigned:')
+      = select_tag('available_fields[]',
+        options_for_select(@edit[:new][:available_fields]),
+        :multiple => true,
+        :class    => "form-control",
+        :style    => "overflow-x: scroll;",
+        :size     => 8,
+        :id       => "available_fields")
+  - if x_active_tree == :ab_tree
```

Comment:
Hi Eric @epwinchell , I have a question: why do we have here this condition `if x_active_tree == :ab_tree` twice? Once in https://github.com/ManageIQ/manageiq-ui-classic/pull/1376/files#diff-fb2d45f8fceb60d60e1267c1d866e824R2, the second time in https://github.com/ManageIQ/manageiq-ui-classic/pull/1376/files#diff-fb2d45f8fceb60d60e1267c1d866e824R12
Is there any reason for that? Thank you! :)

Reply:
@h-kataria  Any ideas?

Reply:
@epwinchell @hstastna if you look closely at the old/refactored code that was removed had piece of code that was not part of if block
https://github.com/ManageIQ/manageiq-ui-classic/pull/1376/files#diff-fb2d45f8fceb60d60e1267c1d866e824L8

looks like same for the second if block as well https://github.com/ManageIQ/manageiq-ui-classic/pull/1376/files#diff-fb2d45f8fceb60d60e1267c1d866e824L47